balancer/pickfirst/pickfirst.go

```diff
@@ -274,10 +534,10 @@ func (p *picker) Pick(balancer.PickInfo) (balancer.PickResult, error) {
 // idlePicker is used when the SubConn is IDLE and kicks the SubConn into
 // CONNECTING when Pick is called.
 type idlePicker struct {
-	subConn balancer.SubConn
+	callback func()
```

Comment:
`callback` is not a very descriptive name. Maybe call it `exitIdle` or `connectFunc` or something to indicate that the responsibility of this field is to start connecting.

Reply:
Changed to `exitIdle`.